app/src/main/res/layout/activity_my_account.xml

```diff
@@ -0,0 +1,266 @@
+
+<!--<android.support.constraint.ConstraintLayout-->
+    <!--android:id="@+id/constraint_layout_PG"-->
+    <!--android:layout_width="match_parent"-->
+    <!--android:layout_height="match_parent">-->
+
+    <!--<TextView-->
+        <!--android:id="@+id/past_games_text"-->
+        <!--android:text="@string/pastGamesText"-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:layout_marginLeft="16dp"-->
+        <!--app:layout_constraintLeft_toLeftOf="@id/constraint_layout_PG"-->
+        <!--app:layout_constraintBottom_toBottomOf="@id/constraint_layout_PG"-->
+        <!--app:layout_constraintTop_toTopOf="@id/constraint_layout_PG"/>-->
+
+    <!--<ImageView-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:src="@mipmap/ic_launcher"-->
+        <!--android:adjustViewBounds="true"-->
+        <!--android:scaleType="fitEnd"-->
+        <!--android:contentDescription="die"-->
+        <!--app:layout_constraintRight_toRightOf="@id/constraint_layout_PG"/>-->
+
+<!--</android.support.constraint.ConstraintLayout>-->
+
+<!--</LinearLayout>-->
+
+<!--<LinearLayout-->
+<!--android:id="@+id/launch_hosted_games_activity_layout"-->
+<!--android:orientation="horizontal"-->
+<!--android:clickable="true"-->
+<!--android:layout_width="match_parent"-->
+<!--android:layout_height="wrap_content"-->
+<!--android:focusable="true"-->
+<!--app:layout_constraintTop_toBottomOf="@id/launch_past_games_activity_layout"-->
+<!--android:layout_marginTop="8dp"-->
+<!--android:layout_marginLeft="16dp"-->
+<!--android:layout_marginRight="16dp"-->
+<!--android:background="@drawable/button_border">-->
+
+<!--<android.support.constraint.ConstraintLayout-->
+    <!--android:id="@+id/constraint_layout_HG"-->
+    <!--android:layout_width="match_parent"-->
+    <!--android:layout_height="match_parent">-->
+
+    <!--<TextView-->
+        <!--android:id="@+id/hosted_games_text"-->
+        <!--android:text="@string/hostedGamesText"-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:layout_marginLeft="16dp"-->
+        <!--app:layout_constraintLeft_toLeftOf="@id/constraint_layout_HG"-->
+        <!--app:layout_constraintBottom_toBottomOf="@id/constraint_layout_HG"-->
+        <!--app:layout_constraintTop_toTopOf="@id/constraint_layout_HG"/>-->
+
+    <!--<ImageView-->
+        <!--android:layout_width="wrap_content"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:src="@mipmap/ic_launcher"-->
+        <!--android:adjustViewBounds="true"-->
+        <!--android:scaleType="fitEnd"-->
+        <!--android:contentDescription="die"-->
+        <!--app:layout_constraintRight_toRightOf="@id/constraint_layout_HG"/>-->
+
+<!--</android.support.constraint.ConstraintLayout>-->
+
+<!--</LinearLayout>-->
+
```

Comment:
Please remove this useless comments. If they're usefull add a TODO/FIXME with an asignee and an explaination